class/defaults.yml

```
@@ -12,3 +12,7 @@ parameters:
         http01:
           ingress:
             class: 'nginx'
+    images:
+      kubectl:
+        image: quay.io/bitnami/kubectl
```

Split image into `registry` and `image`:

```suggestion
        registry: quay.io
        image: bitnami/kubectl
```
and reconstruct the full image string in Jsonnet. This allows easier overriding of the registry (e.g. with some mirror, or for environments which don't have unrestricted access to the internet).